packages/discord.js/src/structures/Webhook.js

```diff
@@ -393,6 +403,22 @@ class Webhook {
     return this.avatar && this.client.rest.cdn.Avatar(this.id, this.avatar, options);
   }
 
+  /**
+   * Whether this webhook is created by a user.
+   * @returns {boolean}
+   */
+  isUserCreated() {
+    return Boolean(this.owner);
+  }
```

Comment:
This isn't accurate - if you truly want to know if it was created by a user and not a bot, you would have to check if the `owner` is a bot too.

Reply:
Not sure why I added this. Is it fine if I remove it? If it's incoming, the user/developer can check themselves, no?